augly/text/transforms.py

```diff
@@ -951,12 +956,24 @@ def apply_transform(
             including its name, the source & dest length, etc. will be appended to
             the inputted list. If set to None, no metadata will be appended or returned
 
-        @param aug_kwargs: kwargs to pass into the augmentation that will override values
-            set in __init__
-
         @returns: the list of augmented text documents
         """
-        return F.simulate_typos(texts, metadata=metadata, **aug_kwargs)
```

Comment:
as @zpapakipos mentioned in the group chat, let's revert this change in `apply_transform` and use `aug_kwargs` instead of writing out each of the arguments (as this is in line for all the other functions in the `text` library now :)